src/lib/components/SecureIframe.js

```diff
@@ -21,7 +21,7 @@ export default function SecureIframe(allProps) {
   const [loaded, setLoaded] = useState(false);
   const [prevMessage, setPrevMessage] = useState(undefined);
   const ref = React.useRef();
-  const sandbox = "allow-scripts";
+  const sandbox = "allow-scripts allow-popups allow-popups-to-escape-sandbox";
```

Comment:
`Keep in mind that the use of these attributes should be carefully considered, as it can introduce potential security risks, especially if you allow scripts to run. You should use them only when you trust the content loaded within the iframe.`

Reply:
cc @matejpribanic  @xilosada  Your call guys if we make this available , withouth it chat links wont be clickable with current logic. I can revert that urls get extracted from the message and then mapped in <a> tags at the end of the message like we had before

Reply:
@xilosada I would like for user to click on the link and it opens in a new tab,  but idk if its security risk its ur call